Parse/src/main/java/com/parse/ParseFileController.java

```diff
@@ -41,7 +41,7 @@ public ParseFileController(ParseHttpClient restClient, File cachePath) {
   /* package */ ParseHttpClient awsClient() {
     synchronized (lock) {
       if (awsClient == null) {
-        awsClient = ParsePlugins.get().newHttpClient();
+        awsClient = ParsePlugins.get().awsClient();
```

Comment:
Yeah, I will go ahead and have the HTTP and File clients be the same builder, just two different client instances to mock the present behavior. The reasoning for having them both be configurable would just be in case we have someone who has two different servers, one for Parse and one that hosts files, and it is possible though unlikely that they could be configured with different HTTPS configurations. But I am all for keeping it limited to the one builder for now.